zeebe/protocol/src/main/java/io/camunda/zeebe/protocol/ZbColumnFamilies.java

```diff
@@ -171,80 +173,92 @@ public enum ZbColumnFamilies implements EnumValue {
   USER_TASK_STATES(80),
   COMPENSATION_SUBSCRIPTION(81),
 
-  PROCESS_DEFINITION_KEY_BY_PROCESS_ID_AND_DEPLOYMENT_KEY(82),
-  DMN_DECISION_KEY_BY_DECISION_ID_AND_DEPLOYMENT_KEY(83),
-  FORM_KEY_BY_FORM_ID_AND_DEPLOYMENT_KEY(84),
+  PROCESS_DEFINITION_KEY_BY_PROCESS_ID_AND_DEPLOYMENT_KEY(82, ColumnFamilyScope.GLOBAL),
+  DMN_DECISION_KEY_BY_DECISION_ID_AND_DEPLOYMENT_KEY(83, ColumnFamilyScope.GLOBAL),
+  FORM_KEY_BY_FORM_ID_AND_DEPLOYMENT_KEY(84, ColumnFamilyScope.GLOBAL),
 
   MESSAGE_CORRELATION(85),
 
-  USERS(86),
+  USERS(86, ColumnFamilyScope.GLOBAL),
   @Deprecated
-  USER_KEY_BY_USERNAME(87),
+  USER_KEY_BY_USERNAME(87, ColumnFamilyScope.GLOBAL),
 
-  CLOCK(88),
+  CLOCK(88, ColumnFamilyScope.GLOBAL),
 
-  AUTHORIZATIONS(89),
-  PERMISSIONS(90),
+  AUTHORIZATIONS(89, ColumnFamilyScope.GLOBAL),
+  PERMISSIONS(90, ColumnFamilyScope.GLOBAL),
 
-  PROCESS_DEFINITION_KEY_BY_PROCESS_ID_AND_VERSION_TAG(91),
-  DMN_DECISION_KEY_BY_DECISION_ID_AND_VERSION_TAG(92),
-  FORM_KEY_BY_FORM_ID_AND_VERSION_TAG(93),
+  PROCESS_DEFINITION_KEY_BY_PROCESS_ID_AND_VERSION_TAG(91, ColumnFamilyScope.GLOBAL),
+  DMN_DECISION_KEY_BY_DECISION_ID_AND_VERSION_TAG(92, ColumnFamilyScope.GLOBAL),
+  FORM_KEY_BY_FORM_ID_AND_VERSION_TAG(93, ColumnFamilyScope.GLOBAL),
 
   @Deprecated
-  AUTHORIZATION_KEY_BY_RESOURCE_ID(94),
+  AUTHORIZATION_KEY_BY_RESOURCE_ID(94, ColumnFamilyScope.GLOBAL),
   @Deprecated
-  OWNER_TYPE_BY_OWNER_KEY(95),
+  OWNER_TYPE_BY_OWNER_KEY(95, ColumnFamilyScope.GLOBAL),
 
-  ROUTING(96),
+  ROUTING(96, ColumnFamilyScope.GLOBAL),
 
   QUEUED_DISTRIBUTION(97),
   RETRIABLE_DISTRIBUTION(98),
   DISTRIBUTION_CONTINUATION(99),
 
-  RESOURCES(100),
-  RESOURCE_VERSION(101),
-  RESOURCE_BY_ID_AND_VERSION(102),
-  RESOURCE_KEY_BY_RESOURCE_ID_AND_VERSION_TAG(103),
-  RESOURCE_KEY_BY_RESOURCE_ID_AND_DEPLOYMENT_KEY(104),
+  RESOURCES(100, ColumnFamilyScope.GLOBAL),
+  RESOURCE_VERSION(101, ColumnFamilyScope.GLOBAL),
+  RESOURCE_BY_ID_AND_VERSION(102, ColumnFamilyScope.GLOBAL),
+  RESOURCE_KEY_BY_RESOURCE_ID_AND_VERSION_TAG(103, ColumnFamilyScope.GLOBAL),
+  RESOURCE_KEY_BY_RESOURCE_ID_AND_DEPLOYMENT_KEY(104, ColumnFamilyScope.GLOBAL),
 
-  ROLES(105),
+  ROLES(105, ColumnFamilyScope.GLOBAL),
 
-  TENANTS(106),
+  TENANTS(106, ColumnFamilyScope.GLOBAL),
 
   USER_TASK_INTERMEDIATE_STATES(107),
 
-  MAPPINGS(108),
+  MAPPINGS(108, ColumnFamilyScope.GLOBAL),
   CLAIM_BY_KEY(109),
 
   USER_TASK_TRANSITION_TRIGGER_REQUEST_METADATA(110),
 
-  GROUPS(111),
+  GROUPS(111, ColumnFamilyScope.GLOBAL),
 
   REDISTRIBUTION(112),
 
-  USERNAME_BY_USER_KEY(113),
-  CLAIM_BY_ID(114),
-  AUTHORIZATION_KEYS_BY_OWNER(115),
+  USERNAME_BY_USER_KEY(113, ColumnFamilyScope.GLOBAL),
+  CLAIM_BY_ID(114, ColumnFamilyScope.GLOBAL),
+  AUTHORIZATION_KEYS_BY_OWNER(115, ColumnFamilyScope.GLOBAL),
 
   BATCH_OPERATION(116),
   PENDING_BATCH_OPERATION(117),
   BATCH_OPERATION_CHUNKS(118),
```

Comment:
❓ Could you check these with the batch operations team? I can see these get populated on all partitions. If they're only related to batch operations on a process instance level it's likely not needed. However, if they are also on a resource level, such as deleting process definitions in a batch, we should ensure we have them on the new partition as well.

Reply:
`BATCH_OPERATION_CHUNKS` is definitely single partition scoped.